packages/at_onboarding_flutter/example/pubspec.yaml

```diff
@@ -38,8 +38,14 @@ dependencies:
   at_backupkey_flutter: ^4.0.8
 
 dependency_overrides:
+  path: 1.8.3
   at_onboarding_flutter:
     path: ../
+  at_client_mobile:
+    git:
+      url: https://github.com/atsign-foundation/at_client_sdk
+      path: packages/at_client_mobile
+      ref: trunk
```

Comment:
can we change the `at_client_mobile` reference to hosted version?

Reply:
Yes, I see that you've deleted it. I've tested and it's working well. Can you merge it? @sachins-geekyants